tests/test_flow/test_api.py

```diff
@@ -623,6 +624,43 @@ def xs_sum(xs):
     assert index_items == [[1, 2], [2, 3]]
 
 
+def test_entity_serialization_exception(builder):
+    @builder
+    def unpicklable_value():
+        def f():
+            return 1
+        return f
+
+    try:
+        builder.build().get('unpicklable_value')
+    except EntitySerializationError as e:
+        # AttributeError is what happens when we try to pickle a function.
+        assert isinstance(e.__cause__, AttributeError)
+
+
+def test_entity_computation_exception(builder):
+    @builder
+    def uncomputable_value():
+        return 1 / 0
+
+    try:
+        builder.build().get('uncomputable_value')
+    except EntityComputationError as e:
+        assert isinstance(e.__cause__, ZeroDivisionError)
+
+
+def test_multiple_compute_attempts(builder):
+    @builder
+    def uncomputable_value():
+        return 1 / 0
+
+    flow = builder.build()
+    with pytest.raises(EntityComputationError):
```

Comment:
nit: can you add a quick comment here why this is tested twice

Reply:
Good idea!